source/geometries/Next100FieldCage.h

```diff
@@ -101,6 +106,9 @@ namespace nexus {
     CylinderPointSampler2020* anode_gen_;
     CylinderPointSampler2020* holder_gen_;
 
+    // Hexagonal Mesh Tool
+    HexagonMeshTools* HexCreator;
```

Comment:
In nexus global variables end with an underscore. Also, we use the convention of lower case letter and `_` for the name of the variables (it may be not completely consistent across the code, but we're trying to follow this rule for new code). 
You can have a look [here](https://github.com/next-exp/nexus/wiki/Coding-conventions-and-style-guide) for the code conventions we're using.

Reply:
Thanks for the reminder, I have updated the variable names where I see them.